### PR TITLE
Changelog v1.14.2

### PR DESCRIPTION
**Pull request by 6543**
as title

### CODE REVIEW

**Review by zeripath — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,54 @@ This changelog goes through all the changes that have been made in each release
...
+  * Issue list alignment tweaks (#15483) (#15766)
+  * Implement delete release attachments and update release attachments' name (#14130) (#15666)
+  * Add placeholder text to deploy key textarea (#15575) (#15576)
+  * Project board improvements (#15429) (#15560)
+  * Repo branch page: label size, PR ref, new PR button alignment (#15363) (#15365)
+* MISC
+  * Fix webkit calendar icon color on arc-green (#15713) (#15728)
+  * Performance improvement for last commit cache and show-ref (#15455) (#15701)
+  * Bump unrolled/render to v1.1.0 (#15581) (#15608)
+  * Backport Add ETag header (#15370) (#15552)
```
Comment (techknowlogick) [inline CHANGELOG.md]: Remove ‘Backport’ from this line

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,54 @@ This changelog goes through all the changes that have been made in each release
...
+  * Defer closing the gitrepo until the end of the wrapped context functions (#15653) (#15746)
+  * Fix some ui bug about draft release (#15137) (#15745)
+  * Only log Error on getLastCommitStatus error to let pull list still be visible (#15716) (#15715)
+  * Move tooltip down to allow selection of Remove File on error (#15672) (#15714)
+  * Fix setting redis db path (#15698) (#15708)
+  * Fix DB session cleanup (#15697) (#15700)
+  * Fixed several activation bugs (#15473) (#15685)
+  * Delete references if repository gets deleted (#15681) (#15684)
+  * Fix orphaned objects deletion bug (#15657) (#15683)
+  * Backport Delete protected branch if repository gets removed (#15658) (#15676)
```
Comment (techknowlogick) [inline CHANGELOG.md]: Same as below

**Review by techknowlogick — APPROVED** [review]
Two minor nits, but otherwise LGTM

File: CHANGELOG.md (hunk 2 above)
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Delete protected branch if repository gets removed (#15658) (#15676)
```

File: CHANGELOG.md (hunk 1 above)
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Add ETag header (#15370) (#15552)
```

**Review by lafriks — APPROVED** [review]

**Comment by zeripath** [comment]
Should we add #15796 to this?

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,54 @@ This changelog goes through all the changes that have been made in each release
...
+  * Resolve panic on failed interface conversion in migration v156 (#15604) (#15610)
+  * Fix missing storage init (#15589) (#15598)
+  * If the default branch is not present do not report error on stats indexing (#15546 & #15583) (#15594)
+  * Fix lfs management find (#15537) (#15578)
+  * Fix NPE on view commit with notes (#15561) (#15573)
+  * Fix bug on commit graph (#15517) (#15530)
+  * Send size to /avatars if requested (#15459) (#15528)
+  * Prevent migration 156 failure if tag commit missing (#15519) (#15527)
+* ENHANCEMENTS
+  * Exponential Backoff for ByteFIFO (#15724) (#15793)
```
Comment (6543) [inline CHANGELOG.md:45]:
```suggestion
  * Display conflict-free merge messages for pull requests (#15773) (#15796)
  * Exponential Backoff for ByteFIFO (#15724) (#15793)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,54 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.14.2](https://github.com/go-gitea/gitea/releases/tag/v1.14.2) - 2021-05-08
+
+* API
+  * Make change repo settings work on empty repos (#15778) (#15789)
+  * Add pull "merged" notification subject status to API (#15344) (#15654)
+* BUGFIXES
+  * Use pulls in commit graph unless pulls are disabled (#15734 & #15740 & #15774) (#15775)
```
Comment (6543) [inline CHANGELOG.md:14]:
```suggestion
  * Ensure that ctx.Written is checked after issues(...) calls (#15797) (#15798)
  * Use pulls in commit graph unless pulls are disabled (#15734 & #15740 & #15774) (#15775)
```

**Comment by 6543** [comment]
🚀

**Comment by 6543** [comment]
-> #15799

**Pull request merged**